oak/server/rust/oak_runtime/src/runtime.rs

```diff
@@ -0,0 +1,136 @@
+use std::prelude::v1::*;
```

Comment:
Can you also add `#![no_std]` at the top, or not yet?

Reply:
My intentions are to control this via feature flags, `runtime/lib.rs` has the following

~~~
#![cfg_attr(not(feature = "std"), no_std)]
#![cfg_attr(not(feature = "std"), feature(alloc))]
~~~

where `std` is a default feature flag in the `Cargo.toml`.

But it won't compile fully with `no_std`, and I plan to address this quickly in follow up PRs